packages/collaboration/src/collaboratorspanel.tsx

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import * as React from 'react';
+
+import { Awareness } from 'y-protocols/awareness';
+
+import { Panel } from '@lumino/widgets';
+
+import { ReactWidget } from '@jupyterlab/apputils';
+
+import { ICurrentUser } from './tokens';
+import { ICollaboratorAwareness } from './utils';
+import { PathExt } from '@jupyterlab/coreutils';
+
+/**
+ * The CSS class added to collaborators list container.
+ */
+const COLLABORATORS_LIST_CLASS = 'jp-CollaboratorsList';
+
+/**
+ * The CSS class added to each collaborator element.
+ */
+const COLLABORATOR_CLASS = 'jp-Collaborator';
+
+/**
+ * The CSS class added to each collaborator element.
+ */
+const CLICKABLE_COLLABORATOR_CLASS = 'jp-ClickableCollaborator';
+
+/**
+ * The CSS class added to each collaborator icon.
+ */
+const COLLABORATOR_ICON_CLASS = 'jp-CollaboratorIcon';
+
+export class CollaboratorsPanel extends Panel {
+  private _currentUser: ICurrentUser;
+  private _awareness: Awareness;
+  private _body: CollaboratorsBody;
+
+  constructor(
+    currentUser: ICurrentUser,
+    awareness: Awareness,
+    fileopener: (path: string) => void
+  ) {
+    super({});
+
+    this._awareness = awareness;
+
+    this.title.label = 'Online Collaborators';
```

Comment:
We need to translate that string - I would advice setting it in the activation function of the plugin rather than here to avoid bring down the translator object.